src/PassKit/PKEnums.cs

```diff
@@ -74,11 +76,24 @@ public enum PKPaymentAuthorizationStatus : long {
 		PinLockout
 	}
 
+	[Deprecated (PlatformName.iOS, 13, 4, message: "Use 'PKSecureElementPassActivationState' instead.")]
+	[Deprecated (PlatformName.WatchOS, 6, 2, message: "Use 'PKSecureElementPassActivationState' instead.")]
+	[Deprecated (PlatformName.MacOSX, 10, 15, 4, message: "Use 'PKSecureElementPassActivationState' instead.")]
 	[Native]
 	public enum PKPaymentPassActivationState : ulong {
 		Activated, RequiresActivation, Activating, Suspended, Deactivated
 	}
 
+	[Watch (6,2), Mac (10,15,4), iOS (13,4)]
```

Comment:
^ afaik PassKit is not available on macOS

Reply:
```suggestion
	[Watch (6,2), iOS (13,4)]
```